packages/svelteui-core/src/components/Chip/Chip.styles.ts

```diff
@@ -0,0 +1,165 @@
+import { createStyles } from '$lib/styles';
+import type { DefaultProps, SvelteUIColor, SvelteUINumberSize, SvelteUISize } from '$lib/styles';
+import type { Component } from '$lib/internal';
+
+export interface ChipProps extends DefaultProps {
+	color?: SvelteUIColor;
+	id?: string;
+	checked?: boolean;
+	disabled?: boolean;
+	value?: string;
+	label?: Component | string;
+	radius?: SvelteUINumberSize | number;
+	size?: SvelteUISize;
+  type?: 'checkbox' | 'radio';
+  variant?: 'outline' | 'filled'
+	transitionDuration?: number;
+	wrapperProps?: { [key: string]: any };
+}
+
+export interface ChipStyleParams {
+	size: SvelteUISize;
+	color: SvelteUIColor;
+  variant: 'outline' | 'filled'
+	radius: SvelteUINumberSize | number;
+	transitionDuration: number;
+}
+
+export const sizes = {
+  xs: 24,
+  sm: 28,
+  md: 32,
+  lg: 36,
+  xl: 40,
+};
+
+const iconSizes = {
+  xs: 10,
+  sm: 12,
+  md: 14,
+  lg: 16,
+  xl: 18,
+};
+
+const padding = {
+  xs: 16,
+  sm: 20,
+  md: 24,
+  lg: 28,
+  xl: 32,
+};
+
+const checkedPadding = {
+  xs: 7.5,
+  sm: 10,
+  md: 11.5,
+  lg: 13,
+  xl: 15,
+};
+
+export default createStyles(
+	(theme, { color, radius, size, transitionDuration, variant }: ChipStyleParams, getRef) => {
+		return {
+      root: {},
+
+      label: {
+        ref: getRef('label'),
+        boxSizing: 'border-box',
+        color: theme.colorScheme === 'dark' ? theme.colors.dark50 : theme.colors.black,
```

Comment:
We normally use the `theme.fn.themeColor` util, for example in this line if would look like `theme.fn.themeColor('dark', 50)`.

For colors with numbers > 100, it can be `theme.fn.themeColor(color, 1)` - example for color100, and for absolute colors like `white` or `black`, the notation is `theme.colors.black.value`.

Sorry, this can be confusing 😅